modules/logging/aws-for-fluent-bit/main.tf

```diff
@@ -0,0 +1,82 @@
+# Create namespace logging
+resource "kubernetes_namespace" "logging" {
```

Comment:
Script down if namespace exist (use get namespace.name logging and if exist = count 0)

Reply:
Done